setup.py

```diff
@@ -15,7 +15,7 @@ def import_requirements(filename) -> list:
     name="immuneML",
     version=Constants.VERSION,
     description="immuneML is a software platform for machine learning analysis of immune receptor repertoires.",
-    long_description=open("README.md").read(),
+    long_description=open("Genmodel_quickstart.md").read(),
```

Comment:
no need to do this, the quickstart will be in the documentation, it doesn't have to be the package description :)